framework/wazuh/tests/util.py

```diff
@@ -10,9 +10,10 @@
 
 
 class InitWDBSocketMock:
-    def __init__(self, sql_schema_file):
+    def __init__(self, sql_schema_file, mitre=False):

```

Comment:
This flag may be omitted in favour of purely checking the query format with regex.